src/K9OCRS/ClientApp/src/areas/applications/Catalog/components/style.scss

```
@@ -15,4 +15,22 @@
     box-shadow: $box-shadow;
     transform: translate(0,-.2rem);
   }
+}
+
+.classThumbnails {
+  height: 250px;
+  width: 100%;
+  object-fit: cover;
+  box-shadow: $box-shadow;
+}
+
+.cropped {
+  width: 250px;
+  height: 150px;
+  overflow: hidden;
+  border: 2px solid gray;
+}
+
+.heroImg {
```

Didn't remove the `.cropped`
```suggestion
}

.heroImg {
```